core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -642,6 +642,7 @@ public void start() {
       if (globalConfiguration.security().authorization().enabled()) {
          globalConfiguration.security().authorization().principalRoleMapper().setContext(new PrincipalRoleMapperContextImpl(this));
       }
+      globalComponentRegistry.start();
```

Comment:
We shouldn't need https://github.com/tristantarrant/infinispan/blob/ISPN-6214/initial_cluster_size/core/src/main/java/org/infinispan/manager/DefaultCacheManager.java#L610 any longer with this added.